### PR TITLE
Fnish basic login

### PR DESCRIPTION
**Pull request by tranxuanhiep**
image

### CODE REVIEW

**Comment by tranxuanhiep** [comment]
ready

**Comment by tranxuanhiep** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.nil?
```
Comment (dothidiemthao) [inline app/helpers/sessions_helper.rb]: dùng current_user.present?

File: test/test_helper.rb [hunk 2]
```diff
@@ -4,6 +4,8 @@
 class ActiveSupport
   class TestCase
     fixtures :all
-    include ApplicationHelper
+    def is_logged_in?
+      !session[:user_id].nil?
```
Comment (dothidiemthao) [inline test/test_helper.rb]: dùng session[:user_id].present?

**Comment by dothidiemthao** [comment]
commented

File: test/test_helper.rb (hunk 2 above)
Comment (tranxuanhiep) [reply test/test_helper.rb]: !session[:user_id].present? hay session[:user_id].present? chị

File: app/helpers/sessions_helper.rb (hunk 1 above)
Comment (tranxuanhiep) [reply app/helpers/sessions_helper.rb]: !current_user.present? hay current_user.present? chị

**Comment by tranxuanhiep** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.present?
```
Comment (dothidiemthao) [inline app/helpers/sessions_helper.rb]: ! .nil? == .present?
Dùng current_user.present? nha

**Comment by dothidiemthao** [comment]
commented

**Comment by tranxuanhiep** [comment]
ready

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**